languages/de.yml

```diff
@@ -0,0 +1,19 @@
+categories: Kategorien
+search: Suche
+tags: Tags
+tagcloud: Tag Cloud
+tweets: Tweets
+prev: zurück
+next: weiter
+comment: Kommentare
+archive_a: Archiv
```

Comment:
Ah ok i got you.
No thats correct. Some german words are the same or very similar sometimes to english.
Some words we keep the speaking in the original language. Like Tweet or Tag Cloud, if you translate them to german no tech guy would know what u mean :)